doc/installation.rst

```diff
@@ -16,7 +16,30 @@ NESTML can then be installed into your local user directory via:
 
 .. code-block:: bash
 
-   pip install --pre nestml
+   pip install nestml
+
+
+Installing the latest release from PPA (Linux)
+----------------------------------------------
+
+NESTML can be installed via the ``apt`` package manager. This requires superuser (sudo) access. First, add the NEST PPA:
+
+.. code-block:: bash
+
+   sudo add-apt-repository ppa:nest-simulator/nest
+
+Then update the index and install the necessary packages:
+
+.. code-block:: bash
+
+   sudo apt update
+   sudo apt install nest python3-nestml
+
```

Comment:
A few python dependencies still need to be installed:

`python3 -m pip install --upgrade odetoolbox pygsl antlr4-python3-runtime==4.10`

Reply:
```suggestion
python3 -m pip install --upgrade odetoolbox pygsl antlr4-python3-runtime==4.10

```